functions/v2/cloudevent/log-cloudevent/src/test/java/functions/AutoLabelInstanceTest.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package functions;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.testing.TestLogHandler;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import java.net.URI;
+import java.util.logging.Logger;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class AutoLabelInstanceTest {
+  private static final Logger logger = Logger.getLogger(AutoLabelInstance.class.getName());
+  private static final TestLogHandler logHandler = new TestLogHandler();
+
+  private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String ZONE = "us-central1-a";
+  private static final String INSTANCE = "lite1";
```

Comment:
Should this be a UUID?

Reply:
This sample only labels the GCE VM. I looked at the create a VM [code](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/compute/cloud-client/src/main/java/compute/CreateInstance.java) and I am not sure I want to have create and delete an instance every test run; therefore I am using an already created VM. Thoughts?

Reply:
Your probably ok, but anything that creates using a fixed name will have problems in our testing environment.